src/protocol/commands/_qXfer_exec_file.rs

```diff
@@ -32,6 +32,6 @@ impl<'a> ParseCommand<'a> for qXferExecFileRead<'a> {
 
         drop(body);
 
-        Some(qXferExecFileRead {pid, offset, length, buf})
+        Some(qXferExecFileRead { pid, offset, length, buf })
```

Comment:
fascinating... I ran `cargo +nightly fmt --all -v -- --check -v`, and it seems none of the files under `commands` are being checked for formatting! I assume this has something to do with the auto-generated nature of the modules...

I should probably file a issue upstream about this.